R/plotEGRET.R

```diff
@@ -0,0 +1,51 @@
+#' Create an EGRET-style plot
+#'  
+#' Use a loadflex object, but create an EGRET plot.
+#' 
+#' @param plot.name the name of the plot the user wants to create. Current
+#' options: ConcTime, ConcQ, FluxTimeDaily, and FluxQ. Each require different
+#' data, see Details.
+#' @param intdat data.frame of interpolation data
+#' @param estdat data.frame of estimation data
+#' @param preds data.frame of load predictions
+#' @param meta loadflex metadata object; it must include constituent,
+#' flow, conc.units, custom (station abbreviation: sta.abbr, and a short
+#' name for the constituent: consti.name)
+#' @param preds.type character specifying if the predictions being used are
+#' concentrations ("Conc") or fluxes ("Flux").
+#' @param ... additional arguments to pass to the plot
+#'
+#' @details ConcTime, ConcQ, and FluxQ require \code{intdat, meta}. FluxTimeDaily
+#' requires \code{intdat, estdat, preds, meta}.
+#'
+#' @importFrom EGRET plotConcTime
+#' @importFrom EGRET plotConcQ
+#' @importFrom EGRET plotFluxTimeDaily
+#' @importFrom EGRET plotFluxQ
+#'
+#' @export
+plotEGRET <- function(plot.name, intdat = NULL, estdat = NULL, preds = NULL, 
```

Comment:
We wrapped the EGRET functions because there were only the four that I could use based on how I made the EGRET object (before your help!). Now it doesn't seem as necessary since we can just show how to convert to EGRET and then plug into EGRET plotting functions directly.